bar-trender-front-end/src/views/FreeDashboardView.js

```diff
@@ -1,4 +1,8 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7173304)
```suggestion
import React, { useEffect, useState } from "react";
```